### ISSUE TITLE
No scrollbar in popup for IE11

### ISSUE THREAD

**Issue by Morkowski** [AzureAD/microsoft-authentication-library-for-js#1397]
## Library
- [x] `msal@1.2.1`
- [ ] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Description
I'm using popup flow for my application and I can't scroll to submit button on IE11. It's becouse scrollbars are deactivated in IE11 by default when calling window.open().

Is there any reason, why you don't add scrollbars='yes' parameter when opening popup?

Here is the code:
`const popupWindow = window.open(urlNavigate, title, "width=" + popUpWidth + ", height=" + popUpHeight + ", top=" + top + ", left=" + left);`

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-core/src/UserAgentApplication.ts#L770

From MDN:
> scrollbars
> For accessibility reasons, it is strongly encouraged to set this feature always to yes.

https://developer.mozilla.org/pl/docs/Web/API/Window/open

**Comment by mvrak** [comment]
@Morkowski I am concerned that the submit button isn't showing up without scrollbars.  Can you add a screenshot?  That is an issue in itself.

**Comment by Morkowski** [comment]
We are integrating with our client and that's how theirs form looks:
![image](https://user-images.githubusercontent.com/12064267/77620535-dde85a80-6f3a-11ea-82b4-833958be913d.png) 

Black boxes are logo and title.

Becouse of described issue I can't reach submit submit button.

pr for this issue: https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/1426
